File: mobile/src/main/java/com/princekumarsingh/infinityitsolution/activity/SplashActivity.java
package com.princekumarsingh.infinityitsolution.activity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.appcompat.app.AppCompatActivity;


/**
 *
 */
public class SplashActivity extends AppCompatActivity {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		int splashscenond = 1000;
		new Handler().postDelayed(() -> {
			Intent splash = new Intent(getApplicationContext(),MainActivity.class);
			startActivity(splash);
			finish();

		}, splashscenond);

	}


	}

File: app/src/main/java/com/infinity_it_solution_assement/activity/SplashActivity.kt
package com.infinity_it_solution_assement.activity

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

/**
 *
 */
class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val splashscenond = 1000
        Handler().postDelayed({
            val splash = Intent(applicationContext, MainActivity::class.java)
            startActivity(splash)
            finish()
        }, splashscenond.toLong())
    }
}